pkg/kclient/deployments.go

```diff
@@ -226,6 +226,9 @@ func (c *Client) CreateDynamicResource(exampleCustomResource map[string]interfac
 		Object: exampleCustomResource,
 	}
 
+	debugOut, _ := json.MarshalIndent(deployment, " ", " ")
```

Comment:
What's the purpose of this code? Can you add a comment?

Reply:
this is for debugging and verbose output.  I don't think that this needs a comment.